src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationPopup/SelectAttributeFromClassifierGuesses.js

```diff
@@ -72,12 +54,26 @@ const SelectAttributeFromClassifierGuesses = ({
     }, [])
     .sort((a, b) => a.label.localeCompare(b.label))
 
-  const _updateSelectedRowOnPointAnnotationChange = useEffect(() => {
-    const rowKeyForPoint = selectedPoint.annotations[0].ba_gr
-    const isPointInARow = existingRowDropdownOptions.some((row) => rowKeyForPoint === row.value)
+  const isPointInARow = existingRowDropdownOptions?.some((row) => rowKeyForPoint === row.value)
+  const selectedExistingRow = isPointInARow ? rowKeyForPoint : ''
 
-    setSelectedExistingRow((prevState) => (isPointInARow ? rowKeyForPoint : prevState))
-  }, [selectedPoint.annotations, existingRowDropdownOptions])
```

Comment:
Basic change is to remove this useEffect and to have the calculation happen in the component scope. It was an unnecessary effect and this seems to have removed the bug.